README.md

```diff
@@ -8,61 +8,79 @@ Head over to the [marketplace](https://www.contentful.com/marketplace/) and foll
 
 ## Adding and maintaining your app in this repo
 
-The Marketplace Partner Apps repo is maintained as a single repository for apps, containing several ecosystem partner apps + the tools needed to build and maintain them. The `apps` folder contains each partner app, and each sub-folder within `apps` should contain all the code necessary to build, test, and deploy your app without references to other apps or folders. All dependencies between apps are indepedent and isolated from each other.
+The Marketplace Partner Apps repo is maintained as a single repository for apps, containing several ecosystem partner apps + the tools needed to build and maintain them. The `apps` folder contains each partner app, and each sub-folder within `apps` should contain all the code necessary to build, test, and deploy your app without references to other apps or folders. All dependencies between apps are independent and isolated from each other.
```

Comment:
Agreed, opted to just remove that sentence since I think the sentence before it captures the essence of what we are trying to get at there, that each app should be its own directory without references to other apps